language/stdlib/modules/script_whitelist.move

```diff
@@ -0,0 +1,16 @@
+address 0x0:
+
+module ScriptWhitelist {
+    use 0x0::LibraConfig;
+
+    struct T { payload: vector<u8> }
```

Comment:
- Comment describing what `payload` looks like  + better name (e.g., `script_hash_list`)?
- What will this look like if we want to allow arbitrary transaction scripts in the future?